### PR TITLE
Fix URL errors on tests.

### PR DESCRIPTION
**Pull request by stdevOzkar**
Fix URL erros on tests.

### CODE REVIEW

File: tests/api_tests.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def setUp(self):
     def test_address_valid(self):
         data = dict(address=r'2GgFvqoyk9RjwVzj8tqfcXVXB4orBwoc9qv')
         response = self.app.get(
-            '/api/v1.0/addresses/{}/isvalid'.format(data),
```
Comment (olemis) [inline tests/api_tests.py]: This is wrong . It has to be `/1.0/api/...` as per Lykke specs .

**Review by olemis — CHANGES_REQUESTED** [review]
`/api/v1.0/...` => `/1.0/api/...` everywhere .

**Comment by stdevOzkar** [comment]
Fixed URL root for exchange_app. According to Lykke specs, we should not have the API version included.

File: tests/api_tests.py (hunk 1 above)
Comment (olemis) [reply tests/api_tests.py]: @stdevOzkar this is the right URL prefix

**Review by olemis — CHANGES_REQUESTED** [review]

**Comment by olemis** [comment]
already fixed in `develop`.

**Pull request closed without merging**